src/index.html

```diff
@@ -19,42 +19,51 @@
     <form
       action="https://mate-academy-form-lesson.herokuapp.com/create-application"
       method="post"
-      class="form-wrapper first-wrapper"
+      class="form-wrapper"
       data-qa="big"
     >
-      <label for="searchOne">
+      <label
+        for="searchOne"
+        class="label-image"
```

Comment:
Think this class name should be just `label` or `search-label` because it points to the inputs by `for` attribute not images